src/entities/user.ts

```diff
@@ -120,6 +122,14 @@ export class User extends BaseEntity {
   @Column({ nullable: true })
   url?: string;
 
+  @Field(_type => String, { nullable: true })
+  @Column({ nullable: true })
+  twitterName?: string;
```

Comment:
I think we can change it to `twitterHandle` and `telegramHandle` or `twitterUsername` and `telegramUsername`

@kkatusic what do you think?